src/useQuery.js

```diff
@@ -30,6 +28,11 @@ module.exports = function useQuery(query, opts = {}) {
 
   return {
     ...state,
-    refetch: () => queryReq({ skipCache: true })
+    refetch: (options = {}) =>
+      queryReq({
+        skipCache: true,
+        updateData: (_, data) => data,
```

Comment:
Minor: We should add a comment here saying why we're doing this

Reply:
Agree, I've added an explanation